source/reference/minio-mc-admin/mc-admin-prometheus-metrics.rst

```diff
@@ -63,146 +74,125 @@ Parameters
 
    The :mc:`alias <mc alias>` of a configured MinIO deployment for which the command prints metrics.
 
+.. mc-cmd:: --api-version
+   :optional:
+
+   To print :ref:`version 3 (v3) <minio-metrics-and-alerts>` metrics, include an ``--api-version v3`` parameter.
+   ``v3`` is the only accepted value.
+
+   Omit ``--api-version`` to print :ref:`version 2 (v2) <minio-metrics-v2>` metrics.
+
+.. mc-cmd:: --bucket
+   :optional:
+
+   For v3 metric types that return bucket-level metrics, specify a bucket name.
+   Use with :mc-cmd:`~mc admin prometheus metrics --api-version`.
+
+   ``--bucket`` works for the following v3 metric types:
+
+   - ``api``
+   - ``replication``
+
+   The following example prints API metrics for the bucket ``mybucket``:
+
+   .. code-block:: shell
+      :class: copyable
+
+      mc admin prometheus metrics ALIAS api --bucket mybucket --api-version v3
+
 .. mc-cmd:: TYPE
    :optional:
 
    The type of metrics to print.
 
-      .. versionchanged:: RELEASE.2023-10-07T15-07-38Z
+      Valid values for metrics version 3 are:
 
-         ``resource`` metrics added
+      - ``api``
+      - ``audit``
+      - ``cluster``
+      - ``debug``
+      - ``ilm``
+      - ``logger``
+      - ``notification``
+      - ``replication``
+      - ``scanner``
+      - ``system``
+
+      If not specified, a ``v3`` command returns all metrics.
 
-      Valid values are:
+      Valid values for metrics version 2 are:
 
       - ``bucket``
       - ``cluster``
       - ``node``
```

Comment:
Node metrics are kinda a subset of cluster? Except when there're not. They don't all rollup correctly into cluster and you supposedly don't need them anyway. Don't really know what to do about this besides hope more people switch to v3.

Reply:
Not much to do, it is a valid value here, right?

Reply:
🤷🏻